### DISCUSSION TITLE
Left click is disabled in "antd Treeselect"

### DISCUSSION THREAD

**Discussion by lexoring** [xyflow/xyflow#2730] [Q&A] [answered]
Hello. I'm using the "Treeselect" component "antd" library and I have this problem. Left click is disabled. Here is the Treeselect component itself from antd https://codesandbox.io/s/8xr5pd?file=/demo.tsx and when you press the left button, the menu comes out.

And here https://codesandbox.io/s/withered-forest-ummcbg?file=/TextUpdaterNode.js does not react on the left button, and if you press the right mouse button, the menu exits.

Although it should respond to the left mouse button. Also, non-left click does not work when you click on the cross in this menu.

The ant.design component itself https://ant.design/components/tree-select
Please, help me! In what there can be a mistake?

[recording (1).webm](https://user-images.githubusercontent.com/74623056/210835835-c3d7e071-ae15-41ac-9d19-dfb53f62d5c5.webm)

**Comment by moklick** [comment, marked as answer]
For interactive elements inside custom nodes it's recommended to add the class name "nodrag". This prevents the dragging behaviour and enables the normal behaviour of that component. I added it for the input and the tree select: https://codesandbox.io/s/optimistic-poincare-0yy7td?file=/TextUpdaterNode.js

**Reply by lexoring** [reply]
You helped me a lot! I am saved!

**Comment by JIAQIA** [comment]
Also works for "Select"

Thanks @moklick  a lot.
